### PR TITLE
Recipe to build the CRAN TPEA package.

### PR DESCRIPTION
**Pull request by boll3**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-tpea/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - r-igraph
+
+test:
+  commands:
+    - $R -e "library('TPEA')"
+
+about:
+  home: https://CRAN.R-project.org/package=TPEA
+  license: GPL (>= 2)
+  summary: 'We described a novel Topology-based pathway enrichment analysis, which integrated the global position of the nodes and the topological property of the pathways in Kyoto Encyclopedia of Genes and Genomes Database. We also provide some functions to obtain the latest information about pathways to finish pathway enrichment analysis using this method.'
```
Comment (bernt-matthias) [inline recipes/r-tpea/meta.yaml]:
```suggestion
  summary: 'Topology-based pathway enrichment analysis, which integrates the global position of the nodes and the topological property of the pathways in Kyoto Encyclopedia of Genes and Genomes Database. Also some functions are provided to obtain the latest information about pathways to finish pathway enrichment analysis.'```

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
Please put this on conda-forge instead, we're trying to keep CRAN packages there when possible.

**Comment by boll3** [comment]
I moved it to conda-forge.

**Pull request closed without merging**